### PR TITLE
Push contracts and seed to NPM

### PR DESCRIPTION
**Pull request by jksf**
This PR depends on #32. Do not merge before.

# Usage examples

### `package.json`
```json
  "scripts": {
    "seed": "signhash-seed",
    "seed-explore": "npm explore signhash-contracts -- npm run seed:js"
  },
  "dependencies": {
    "signhash-contracts": "^0.2.0"
  }
```

`seed` and `seed-explore` do exactly the same

### `example.js`
```javascript
const signhash = require('signhash-contracts');
console.log(signhash.SignHashContract.abi);
```

### CODE REVIEW

File: package.json
Comment (biern) [inline package.json]: So, unfortunately, removing the `migrate` and moving `truffle` to `devDependencies` breaks the webapp build pipeline. Webapp relies on being able to run `contracts:migrate` as a single command. When truffle is in dev dependencies it's not installed when installing `signhash-contracts` as a dependency and thus it is impossible to run migrations via CLI.

**Review by biern — CHANGES_REQUESTED** [review]

**Comment by jksf** [comment]
Fixed compatibility issues with webapp in #32. I rebased this PR to include changes from #32 and published new NPM package (`0.2.1`). Ready to merge right after #32.

**Pull request merged**
